justfile

```diff
@@ -22,6 +22,11 @@ deploy-sequencer:
 deploy-geth:
   kubectl apply -k kubernetes/geth
 
+faucet-default := 'local'
+
+deploy-faucet type=faucet-default:
```

Comment:
Should we add to README and maybe to `deploy-astria-local`?

Reply:
Yeah, I don't have a strong opinion either way. I can see it being useful to just always deploy with "astria-local" for now.

I think if we add more tools though we should split them into a separate command like `astria-local-with-tools` or something.